### PR TITLE
[CONTEXT-234] New Visualization Graph for Assets

### PR DESCRIPTION
**Pull request by ghost**
image

image

### CODE REVIEW

**Comment by anderscognite** [comment]
Can you add some screenshots to this PR as well?

**Comment by anderscognite** [comment]
Does this PR contain changes from the previous PR? I seem to recognize some of the things here...

**Comment by dlcognite** [comment]
@anderscognite right it included the add global state stuff, here is a cleaner one

File: src/containers/TreeViewer.tsx [hunk 1]
```diff
@@ -0,0 +1,315 @@
+import React, { Component } from 'react';
```
Comment (AustinMoninger) [inline src/containers/TreeViewer.tsx]: Would `AssetViewer` or `AssetTreeViewer` be a more apt name since we will have an equivalent one for relationships?

Comment (dlcognite) [reply src/containers/TreeViewer.tsx]: good point, making change now :)

**Review by anderscognite — APPROVED** [review]
LGTM!

**Review by astnexperiments — APPROVED** [review]
Looks good to me

**Pull request merged**